ZukoAzula2016/src/org/team1540/zukoazula/DriveCode.java

```diff
@@ -39,6 +39,7 @@
     private static final Behavior teleop = behaviors.addBehavior("Teleop", FRC.inTeleopMode());
     private static final BooleanCell pitModeEnable = new BooleanCell();
     private static final Behavior pit = behaviors.addBehavior("Pit Mode", pitModeEnable.andNot(FRC.isOnFMS()));
+    private static final Behavior challenge = behaviors.addBehavior("Challenge Brake", ChallengeBrake.driveBrake);
```

Comment:
This behavior acts exactly the same as pit mode. Is there a reason to have them be different?